.flake8

```diff
@@ -14,6 +14,10 @@ max-line-length = 88
 per-file-ignores =
   # For tests, disable type annotation and docstring linting.
   tests/*: ANN D DAR
+  # D enables docstrings warnings from pydocstyle.
+  <!-- Docstrings are mentioned in the original source of the imported functions. -->
+  xrpl/asyncio/clients/async_client.py: D102
+  xrpl/clients/sync_client.py: D102
```

Comment:
Why were linter settings changed?

Reply:
The docstrings are already defined in the respective files they are being imported from, so I chose to not add docstrings and supress the warning

Reply:
If you generate docs locally, do they show up correctly on the methods?